### ISSUE TITLE
Bump libsecp256k1 to 0.5.0

### ISSUE THREAD

**Issue by s3krit** [paritytech/polkadot#3490]
To resolve https://rustsec.org/advisories/RUSTSEC-2021-0076.html we should bump libsecp256k1 to v0.5.0.
cc @sorpaas 

https://docs.rs/libsecp256k1/0.5.0/libsecp256k1/index.html

**Comment by bkchr** [comment]
Duplicate of https://github.com/paritytech/substrate/issues/9356

**Comment by trevor-crypto** [comment]
How will the polkadot repo bump the `libsecp256k1` dependencies to `0.6.0`? Is there a script that automatically looks at substrate and uses what it uses?

**Comment by bkchr** [comment]
Yeah I didn't realized that we use it directly in Polkadot. Sorry! I thought we only do it through Substrate.

**Comment by rphmeier** [comment]
Requires libp2p-core update as well.

**Comment by trevor-crypto** [comment]
Will this allow for my tests to pass here: https://github.com/paritytech/substrate/pull/9391 ?

**Comment by ordian** [comment]
@trevor-crypto your tests will pass when we merge #3477
